### ISSUE TITLE
Blank internet page in authorize() logic

### ISSUE THREAD

**Issue by thebrianbug** [devdennysegura/flutter-auth0#15]
I followed the instructions to use the package, but after configuration, I found that the auth0 page wouldn't load (it showed a blank page). The /authorize route appears to go through successfully, however, the /login route responds with an error:

```
Uncaught DOMException: Failed to execute 'atob' on 'Window': The string to be decoded is not correctly encoded.
```

I've tried to work around this on several new projects with flutter create for a blank slate. I've tried the flutter create both with an without --androidx support, nothing has worked. How can we get past this issue?

@devdennysegura

**Comment by gregertw** [comment]
I'm afraid devdennysegura seems to have abandoned the project :( 
You can try my fork, which has been updated to some of the latest tooling and libs:
https://github.com/gregertw/flutter-auth0
Also see how it is used in this post: https://medium.com/@gregertw/a-production-quality-flutter-starter-app-878801cb1d90?source=friends_link&sk=49d9a1b55da9fb325cccd6cf9f5fe6be

**Comment by devdennysegura** [comment]
this error was fixed on version 0.3.0, release today!
